src/span_ext.rs

```diff
@@ -133,6 +133,25 @@ pub trait OpenTelemetrySpanExt {
     /// app_root.set_attribute("http.request.header.x_forwarded_for", "example");
     /// ```
     fn set_attribute(&self, key: impl Into<Key>, value: impl Into<Value>);
+
+    /// Sets an OpenTelemetry status for this span.
```

Comment:
Nits: keep the first line in a separate paragraph, which improves formatting in rustdoc. Add an empty line before the `# Examples` header and discard the empty line at the end of the code sample.

Reply:
updated.